emulator/blockchain.go

```diff
@@ -735,6 +750,7 @@ func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPubli
 	options = append(options,
 		fvm.WithInitialTokenSupply(supply),
 		fvm.WithRestrictedAccountCreationEnabled(false),
+		fvm.WithSetupEVMEnabled(true),
```

Comment:
Should this use the configuration option?

Reply:
Yes, thanks for catching this.